avm/src/daemon/daemon.rs

```diff
@@ -24,6 +24,32 @@ use crate::vm::run::run;
 pub type DaemonResult<T> = std::result::Result<T, Box<dyn Error + Send + Sync>>;
 
 pub static CLUSTER_SECRET: OnceCell<Option<String>> = OnceCell::new();
+pub static DAEMON_PROPS: OnceCell<DaemonProperties> = OnceCell::new();
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Debug, Serialize)]
+pub struct DaemonFileB64 {
+  pub name: String,
+  pub content: String,
+}
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Debug, Serialize)]
+pub struct DaemonFilesB64 {
+  pub Dockerfile: Option<DaemonFileB64>,
+  pub appTarGz: Option<DaemonFileB64>,
+  pub envVariables: Option<DaemonFileB64>,
+}
```

Comment:
This struct should probably not exist and `filesB64` is just a `Vec< DaemonFileB64>`?

Reply:
Better if it was a `HashMap<String, String>` where the key is the file name and the value is the file contents base64 encoded. Then we can support *any* files needed.

Reply:
I'll try this with the `HashMap`, did not thought about it the first time. This way we remove the struct complexity I'm adding.